### ISSUE TITLE
Playlist Not Showing Up

### ISSUE THREAD

**Issue by seabass707** [Technach/MusicToTHUGPro#1]
Files are where they should be,but the playlist doesn't show up in game. I used mp3 files that when converted turn into a couple kbs so I don't know if thats the issue.

**Comment by Technach** [comment]
@seabass707 You closed this so I'm assuming you resolved the problem but still curious as to what the problem was. If it's still a problem, could you provide the generated JSON on pastebin and tell me what version you are using? I don't maintain this anymore but I'll try and fix the problem

**Comment by seabass707** [comment]
So I had like a list of 50 songs in a folder I wanted to put into the
game,and I followed the instructions from a YouTube video,but when I went
into Thug Pro to listen to my songs, my soundtrack never showed up. I
thought maybe it was the amount of songs so I divided them up into 5
folders of 10 songs and followed instructions with all of them,but one of
the folders still didn't work. I took out a song,and it began working,and I
used the program again and it worked. I'm assuming that song was messing
the program up because it was just the song title,but it didn't have an
album, artist or other name featured on the file. Once I edited those in
with audacity that song began working to. I'm assuming somewhere in the
instructions that was mentioned for it to work,but I skipped it since I was
following the Youtube video.
